ci/kokoro/windows/build-32.bat

```diff
@@ -24,7 +24,7 @@ bazel version
 bazel shutdown
 
 REM Configure the environment to use MSVC 2019 and then switch to PowerShell.
-call "c:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars32.bat"
+call "c:\Program Files (x86)\Microsoft Visual Studio\%MSVC_VERSION%\Community\VC\Auxiliary\Build\vcvars64.bat"
```

Comment:
I assume `vcvars64.bat` is intentional for `build-32.bat`

Reply:
Good catch. No, it was not.